src/components/tableRow.jsx

```diff
@@ -112,7 +113,11 @@ class TableRow extends Component<Object> {
     }
 
     return (
-      <Table.Row negative={isError} className={basic === "very" ? classes.veryBasicGrid : ""}>
+      <Table.Row
+        negative={isError}
+        onClick={() => onRowClick(data)}
```

Comment:
What happens when the `onRowClick` function is null or undefined. This will throw  an error. Can you define a local function in this class and call that from the onClick event. That function can call the prop if it is defined or do nothing otherwise.